GitCommands/Settings/AppSettings.cs

```diff
@@ -19,6 +19,14 @@
 
 namespace GitCommands
 {
+    public enum CommitInfoPosition
```

Comment:
Could you please move this and others into own files, e.g., AppSettings.CommitInfoPosition.cs

Reply:
Done.